tests/01-api.js

```diff
@@ -120,4 +121,55 @@ describe('bedrock-messages-client API requests', function() {
       });
     });
   });
+
+  describe('start API', function() {
+    before(function() {
+      helpers.removeCollections(function() {
+      });
+    });
```

Comment:
removeCollections is definitely an async function.  So it should be...

before(function(done) {
  // done should be passed as the callback to the final operation.
  asyncThing(done);
}

On Mon, Apr 18, 2016 at 1:52 PM, Omar Malik <notifications@github.com>
wrote:

> In tests/01-api.js
> <https://github.com/digitalbazaar/bedrock-messages-client/pull/1#discussion_r60103300>
> :
>
> > @@ -120,4 +121,55 @@ describe('bedrock-messages-client API requests', function() {
> >        });
> >      });
> >    });
> > +
> > +  describe('start API', function() {
> > +    before(function() {
> > +      helpers.removeCollections(function() {
> > +      });
> > +    });
>
> before(helpers.removeCollections); seems to work as intended.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/digitalbazaar/bedrock-messages-client/pull/1/files/2da224b1f14a87aa592522ffc89a1dd75c569e7e#r60103300>
>



-- 
*Matt Collier*